runconfig/fixtures/container_config_1_14.json

```diff
@@ -26,5 +26,8 @@
      "ExposedPorts":{
              "22/tcp": {}
      },
-     "RestartPolicy": { "Name": "always" }
+     "RestartPolicy": { "Name": "always" },
+	 "HostConfig": {
+       "NetworkMode": "default"
+     }
```

Comment:
I don't quite understand this change. Can you please help understand what this is about ?
the "default" implicit mode is introduced recently. The default used to be "bridge".
I assume this is required for backward compatibility check. But isnt it true that the current code should work without the need for this particular change ?